_studio/shared/include/libmfx_core_hw.h

```diff
@@ -18,17 +18,14 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-#include "mfx_common.h"
-
-
 #ifndef __LIBMFX_CORE__HW_H__
 #define __LIBMFX_CORE__HW_H__
 
-#include "umc_va_base.h"
+#if defined(MFX_VA)
 
-mfxU32 ChooseProfile(mfxVideoParam * param, eMFXHWType hwType);
+mfxU32 ChooseProfile(mfxVideoParam const* param, eMFXHWType hwType);
```

Comment:
you removed include of mfx_common.h, but use mfxU32 and eMFXHWType which are nowhere defined. This means this header itself will need to be placed correctly in include list. Can you add minimal set of headers to make this header self contained?